soc/arm/st_stm32/stm32g0/Kconfig.defconfig.stm32g071rb

```diff
@@ -0,0 +1,22 @@
+# Kconfig - ST Microelectronics STM32G071RB MCU
+#
+# Copyright (c) 2019 Philippe Retornaz <philippe@shapescale.com>
+# Copyright (c) 2019 ST Microelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32G071RB
```

Comment:
Use `SOC_STM32G071XX`
We don't need to introduce additional information (RB) here. This would prevent code factorization